simple_history/models.py

```diff
@@ -470,12 +470,16 @@ def get_meta_options(self, model):
         return meta_fields
 
     def post_save(self, instance, created, using=None, **kwargs):
+        if not getattr(settings, "SIMPLE_HISTORY_ENABLED", True):
+            return
         if not created and hasattr(instance, "skip_history_when_saving"):
             return
         if not kwargs.get("raw", False):
             self.create_historical_record(instance, created and "+" or "~", using=using)
 
     def post_delete(self, instance, using=None, **kwargs):
+        if not getattr(settings, "SIMPLE_HISTORY_ENABLED", True):
```

Comment:
How would this handle bulk history update and create?

Reply:
Thanks for the comment, Fixed.